core/services/chainlink/application.go

```diff
@@ -185,7 +187,22 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	keyStore := opts.KeyStore
 	restrictedHTTPClient := opts.RestrictedHTTPClient
 	unrestrictedHTTPClient := opts.UnrestrictedHTTPClient
-	registry := capabilities.NewRegistry(globalLogger)
+
+	var registry commontypes.CapabilitiesRegistry
+	if cfg.Capabilities().Registry().Enabled() {
+		registry = capabilities.NewRegistry(globalLogger)
```

Comment:
Hmmmm... this is problematic: I think the capability registry should always be enabled as it's a core service.